tools/travis-test.sh

```diff
@@ -35,6 +30,10 @@ werrors+="-Werror=nonnull -Werror=pointer-arith"
 
 setup_base()
 {
+  # use default python flags but remoge sign-compare
+  sysflags="$($PYTHON -c "from distutils import sysconfig; \
+    print (sysconfig.get_config_var('CFLAGS'))")"
```

Comment:
I think this would look better if indented to match the first quotation, also with `(...)` instead of line continuation, just use two strings.

Reply:
NVM, is shell script.